client/lib/after-layout-flush/index.ts

```diff
@@ -65,17 +73,18 @@ export default function afterLayoutFlush( func ) {
 				timeoutHandle = undefined;
 				timeoutFunc.apply( callThis, callArgs );
 			}, 0 );
-		};
+		} as T;
 	};
 
 	// if RAF is not supported (in Node.js test environment), the wrapped function
 	// will only set a timeout.
-	let wrappedFunc = scheduleTimeout( func );
+	let wrappedFunc: T = scheduleTimeout( func );
```

Comment:
why do we _always_ create the timeout function even when `requestAnimationFrame()` is available?

Reply:
The correct flow is to go through both. The article linked to in the file explains why, exactly, but it basically boils down to browser rendering timing quirks.

When `requestAnimationFrame` isn't available, we do a best effort of running through just `scheduleTimeout`, which is better than nothing.